pkg/lib/web/static_asset_resolver.go

```diff
@@ -57,9 +58,15 @@ func (r *StaticAssetResolver) StaticAssetURL(id string) (string, error) {
 	asset := result.(*StaticAsset)
 
 	assetPath := strings.TrimPrefix(asset.Path, StaticAssetResourcePrefix)
+
+	b, err := readcloserthunk.Performance_Bytes(asset.ReadCloserThunk)
+	if err != nil {
+		return "", err
+	}
+
 	// md5 is used to compute the hash in the filename for caching purpose only
 	// nolint:gosec
-	hash := md5.Sum(asset.Data)
+	hash := md5.Sum(b)
```

Comment:
Instead of reading the whole thing into memory, hash.Hash is actually a io.Writer. That means we can write something like 

```golang
func Hash(rct ReadCloserThunk, hash hash.Hash) ([]byte, err) {
  rc, err := rct()
  if err != nil {
    return nil, err
  }
  defer rc.Close()

  err = io.Copy(hash, rc)
  if err != nil {
    return nil, err
  }
  
  return hash.Sum(nil), nil
}
```

Add this utility function to readcloserthunk package. Test this function by comparing the result of doing in the original way. That is, we want to make sure the new way of calculating hash is no different from call `md5.Sum(allBytes)`